scene/2d/tile_map.cpp

```diff
@@ -381,7 +381,7 @@ void TileMap::update_dirty_quadrants() {
 			Ref<Texture2D> tex = tile_set->tile_get_texture(c.id);
 			Vector2 tile_ofs = tile_set->tile_get_texture_offset(c.id);
 
-			Vector2 wofs = _map_to_world(E->key().x, E->key().y);
+			Vector2 wofs = _map_to_world(Vector2i(E->key()));
```

Comment:
That's maybe work for another commit, but it might be worth considering if we shouldn't use `Vector2i` directly instead of `PosKey` (but `PosKey` seems to use `int16_t` so I guess it's better for memory?).

Reply:
I think it would make sense to ditch `PosKey` just for simplicity's sake. I can't think of any use case for tiles >= 32768 tile-sizes away from the origin, but I don't see why there needs to be an arbitrary limit, and the memory benefits are negligible.

Worth discussing with @reduz I suppose, and done in another PR.